### PR TITLE
Hud small fixes

### PR DESCRIPTION
**Pull request by pedromdpereira**
## Changelog:
- Sidebar background expanded to fit Unions tab:
image

- [#864] Theologian Trait now inclues holy light icon, a totem and elune moon:
image

Amplified (Looks awful lol):
image

## How to test:
- Check Trait icon in both lifestyle tree and Char information.
- Check if the sidebar looks good

### CODE REVIEW

File: gui/hud.gui [hunk 1]
```diff
@@ -616,7 +616,7 @@ widget = {
 			}
 
 			icon = {
-				size = { 100% 100% }
+				size = { 100% 110% }
```
Comment (zumbak04) [inline gui/hud.gui:619]: It changes the size of the icon but doesn't affect the entire tab.
For example, it doesn't move the Royal Court button down.
You have to change the size of `main_tabs` widget.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: gui/hud.gui (hunk 1 above)
Comment (pedromdpereira) [reply gui/hud.gui]: You are right. I didn't test it with Royal Court on. 
With the background kept at 100% and changing only the main_tabs widget the result is much more acceptable.
image

Thank you!

**Comment by pedromdpereira** [comment]
image

@MemeWizard , Thank you for the new Icon, it's great!

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**